### PR TITLE
[DOC-3185] Add in-cluster monitoring instructions to SMP monitoring options

### PR DESCRIPTION
**Pull request by brian-f-harness**
Add in-cluster monitoring instructions to SMP monitoring options

For reviewers, [a preview](https://6481f24b2c10850c8f24d5cb--harness-developer.netlify.app/docs/self-managed-enterprise-edition/monitor-harness-on-prem) is available.

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by sarthak-harnessio — APPROVED** [review]
This architecture will be replacing the first document. Looks good to me, with all the changes. Let's publish the doc

**Review by kapilgarg1996 — APPROVED** [review]

**Review by pratmit — DISMISSED** [review]

File: docs/self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 1]
```diff
@@ -12,179 +12,203 @@ import Tabs from '@theme/Tabs';
...
 
-## Configure metrics and ingress rules
+</figure>
 
-Follow the steps below on the Kubernetes cluster where you deploy your Harness instance:
+## Set up Prometheus in-cluster
+
+To set up Prometheus in-cluster, do the following:
+
+1. Install your Prometheus server using a Prometheus operator the same cluster as your Harness services and databases.
```
Comment (aimurphy) [inline docs/self-managed-enterprise-edition/monitor-harness-on-prem.md:41]:
```suggestion
1. Install your Prometheus server using a Prometheus operator in the same cluster as your Harness services and databases.
```

missing word

**Review by aimurphy — APPROVED** [review]
Approved with the assumption you'll review my comments.

File: docs/self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 2]
```diff
@@ -12,179 +12,203 @@ import Tabs from '@theme/Tabs';
...
+      prometheus.io/port: '9187'
+      prometheus.io/scrape: 'true'
+global:
+  monitoring:
+    enabled: true
+    port: 8889
+    path: /metrics
+```
+
+:::info note
```
[the remark this answers is not in the document]
Comment (brian-f-harness) [reply docs/self-managed-enterprise-edition/monitor-harness-on-prem.md:207]: updated

**Pull request merged**